packages/tools/client-debugger/client-debugger-view/src/DevtoolsView.tsx

```diff
@@ -438,10 +440,37 @@ function ContainersMenuSection(props: ContainersMenuSectionProps): React.ReactEl
 		);
 	}
 
-	// TODO: add button to refresh list of containers
 	return (
-		<MenuSection header="Containers" key="container-selection-menu-section">
-			{containerSectionInnerView}
-		</MenuSection>
+		<div>
+			<ContainerMenuSection header="Containers">{RefreshButton()}</ContainerMenuSection>
+			<MenuSection header="" key="container-selection-menu-section">
+				{containerSectionInnerView}
+			</MenuSection>
+		</div>
+	);
+}
+
+/**
+ * An refresh button to retrive the latest list of containers.
+ */
+function RefreshButton(): React.ReactElement {
+	const messageRelay = useMessageRelay();
+
+	function handleRefreshClick(): void {
+		// Query for list of Containers
+		messageRelay.postMessage(getContainerListMessage);
+		return;
+	}
+
+	const transparentButtonStyle = {
+		backgroundColor: "transparent",
+		border: "none",
+		cursor: "pointer",
+	};
+
+	return (
+		<button onClick={handleRefreshClick} style={transparentButtonStyle} aria-label="Refresh">
```

Comment:
Nit: To be extra specific for accessibility:
```suggestion
		<button onClick={handleRefreshClick} style={transparentButtonStyle} aria-label="Refresh Containers list">
```